README.md

```diff
@@ -1,2 +1,83 @@
 # Reps-Mobile
-reps.mozilla.org data on mobile
+
+<p align="center">
+  <a href="https://flutter.dev/">
+    <img alt="Flutter" src="https://flutter.dev/assets/flutter-lockup-4cb0ee072ab312e59784d9fbf4fb7ad42688a7fdaea1270ccf6bbf4f34b7e03f.svg" width="170" />
+  </a>
+</p>
+<h1 align="center">
+ Flutter Starter 
+</h1>
+
+Kick off your project with this boilerplate. This starter will help to install  the main flutter configuration files along with the brief introduction to Mozilla Reps mobile application.
+
+## 📝 Description
+
+    reps.mozilla.org data on mobile
+
+## 🚀 Quick start
+
+1.  **Install Flutter on your machine**
```

Comment:
I see that there are too many links forwarding towards https://flutter.dev/docs/get-started in the entire Readme. If all the steps have a generalized link, just add a reference to the general link. There is no need to specify these links and steps all the time which unnecessarily increases the length of Readme.

Reply:
That is what I was suggesting you but as you all said I kept various links so kindly suggest what format is appropriate. The necessary changes will be carried out.

Reply:
Sorry for creating the misunderstanding @arps18. What I was suggesting is that you need step by step installation guidelines for the project/app and not flutter. Please understand the difference between the two, using installation steps for flutter will not help new contributors in understanding how to install the app. A step-by-step instructions on how to clone the app properly and how to run on a local machine should be listed. If you are able to provide links of these steps, then it is fine. I hope I have made myself clear now, if not please feel free to ask anything.